src/vue/modules/movements-top-bar/index.vue

```diff
@@ -233,8 +233,15 @@ export default {
   }
 }
 
+@media (max-width: $sidebar-hide-bp) {
```

Comment:
use @include respond-to-custom($sidebar-hide-bp) {

Reply:
changed

Reply:
changed